### PR TITLE
🌐 feat(i18n): add Odia language

### PR DESCRIPTION
**Pull request by soumendrak**
## Summary

- [Odia Language](https://en.wikipedia.org/wiki/Odia_language) support added

## Changes

- `or.toml` file added

### Accessibility

Not Applicable

### Type of change

- [ ] Bug fix (fixes an issue without altering functionality)
- [x] New feature (adds non-breaking functionality)
- [ ] Breaking change (alters existing functionality)
- [x] UI/UX improvement (enhances user interface without altering functionality)
- [ ] Refactor (improves code quality without altering functionality)
- [x] Documentation update
- [ ] Other (please describe below)

---

## Checklist

- [x] I have verified the accessibility of my changes
- [x] I have tested all possible scenarios for this change
- [ ] I have updated `theme.toml` with a sane default for the feature
- [x] I have made corresponding changes to the documentation:
  - [ ] Updated `config.toml` comments
  - [ ] Updated `theme.toml` comments
  - [x] Updated "Mastering tabi" post in English
  - [x] (Optional) Updated "Mastering tabi" post in Spanish
  - [x] (Optional) Updated "Mastering tabi" post in Catalan

### CODE REVIEW

File: i18n/or.toml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+# https://welpo.github.io/tabi/blog/shortcodes/#multilingual-quotes
+show_original_quote = "ମୂଳ ଉଦ୍ଧୃତ ଦେଖାନ୍ତୁ"
+show_translation = "ଅନୁବାଦ ଦେଖାନ୍ତୁ"
+open_quotation_mark = "“"
+close_quotation_mark = "”"
+
+# Translations for stylised Atom feed.
+# https://welpo.github.io/tabi/atom.xml
+# Must contain "About Feeds"; it will become a link.
+about_feeds = "ଏହା ଏକ ୱେବ୍ ଫିଡ୍, ଯାହାକୁ ଆଟମ୍ ଫିଡ୍ ମଧ୍ୟ କୁହାଯାଏ। ଠିକଣା ବାରରୁ URL କୁ ଆପଣଙ୍କ ନ୍ୟୁଜ୍ ରିଡରରେ କପି କରି ସବ୍ସ୍କ୍ରାଇବ୍ କରନ୍ତୁ। ଅଧିକ ଜାଣିବା ଏବଂ ଆରମ୍ଭ କରିବାକୁ ଫିଡ୍ ବିଷୟରେ ଯାଆନ୍ତୁ। ଏହା ମାଗଣା ଅଟେ ।"
```
Comment (welpo) [inline i18n/or.toml:91]: This string needs to contain "`About Feeds`" exactly as is. This becomes a link to the explanation. See [here](https://welpo.github.io/tabi/atom.xml)

**Review by welpo — CHANGES_REQUESTED** [review]
Thank you so much!

Could you please take a look at the comments?

File: i18n/or.toml [hunk 2]
```diff
@@ -0,0 +1,93 @@
+language_name = "Odia"  # Shown in language picker for multi-language sites.
```
[the remark this answers is not in the document]
Comment (soumendrak) [reply i18n/or.toml]: ok, modified.

File: content/blog/faq-languages/index.es.md [hunk 3]
```diff
@@ -35,6 +35,7 @@ tabi admite los siguientes idiomas:
 - Inglés
 - Italiano
 - Japonés
+- Odia
```
[the remark this answers is not in the document]
Comment (soumendrak) [reply content/blog/faq-languages/index.es.md:38]: `Oriya` is the [old name](https://en.wikipedia.org/wiki/Odia_language) of the language. Can you please share me the wikipedia links I will try to update the pages for both the languages.
image

[the remark this answers is not in the document]
Comment (welpo) [reply content/blog/faq-languages/index.es.md:38]: Interesting!

- Spanish wiki: https://es.wikipedia.org/wiki/Idioma_oriya has "Oriya" in the URL, but uses Oriya and Odia interchangeably throughout the article
- Catalan wiki: https://ca.wikipedia.org/wiki/Oriya has "Oriya" in the URL

Let's leave the updated name in the tabi docs

**Comment by welpo** [comment]
Thank you for your time and effort!

Merging~

**Review by welpo — APPROVED** [review]

**Pull request merged**